api/server/sdk/alerts.go

```diff
@@ -33,10 +33,10 @@ import (
 // AlertsServer implements api.OpenStorageAlertsServer.
 // In order to use this server implementation just have
 // AlertsServer pointer properly instantiated with a valid
-// alerts.Reader.
+// alerts.FilterDeleter.
 type AlertsServer struct {
-	// Reader holds pointer to alerts Reader
-	Reader alerts.Reader
+	// FilterDeleter holds pointer to alerts FilterDeleter
+	FilterDeleter alerts.FilterDeleter
```

Comment:
Please change this to non-exported.

Reply:
ideally i would like this to be unexported, but i need to use it elsewhere (px test). One way to achieve would be to create an exported NewAlertsServer(filterDeleter alerts.FilterDeleter) func that would create an instance of interface defined in proto file.